lib/stripe/subscriptions/usage.ex

```diff
@@ -33,7 +33,7 @@ defmodule Stripe.SubscriptionItem.Usage do
   Creates a usage record for a specified subscription item id and date, and fills it with a quantity.
   """
   def create(id, params, opts \\ [])
-
+  
```

Comment:
This is creating a mix errror as well.  We should make sure it doesn't show up in the PR.

`git checkout origin/master lib/stripe/subscriptions/usage.ex`